### ISSUE TITLE
Change model input to be single channel

### ISSUE THREAD

**Issue by kevinhughes27** [kevinhughes27/TensorKart#3]
The color information is likely useless. I am not sure why they bother to use it in the nvidia paper.

**Comment by millerda** [comment]
Is there still interest in doing this?

**Comment by kevinhughes27** [comment]
There sure is!

I see two main benefits to this:
* performance - less input data to the model means it should calculate faster
* simplicity - the color data likely doesn't add any real value and this will make the model a bit simpler looking

**Comment by millerda** [comment]
Okay, I have adjusted the convolutional layer sizes (I left the fully connected layers unchanged) to work with a single channel and added in a line to utils.resize_image that converts the image to grayscale (used when preparing data).

I am unable to get the new gym environment-based play.py to work however, so I am having trouble testing it realtime. I have tried using the old play.py that didn't use the gym environment, and it does seem to work (meaning the model evaluates with the grayscale input) but I am having other issues related to using a non-Xbox 360 controller which has different axis ranges and so it is a little hard to evaluate (Just so I can compare, what are your axes ranges that are recorded? When I override it no longer accepts the controller inputs, I suspect because they are so small when changed to int they are just 0).

Until I get my other possible issues sorted out and am able to test this more fully I don't want to make a pull request, but if you want to try it out, the changes are merged in my fork.

**Comment by kevinhughes27** [comment]
> what are your axes ranges that are recorded? 

It looks like mine goes from -1.0 to 1.0

I should be able to check it and test it for you! I'm pretty busy right now so it might not be for a bit.

**Comment by millerda** [comment]
No worries! My axis values are all positive and between 0 and ~0.00024, so that could definitely be causing the issues I am seeing with the trained model. Let me know how your performance is, I did notice smaller data sizes (which makes sense) and it seemed to train faster as well.

**Comment by bzier** [comment]
Hey @millerda, what kinds of issues are you having with the gym environment? Was it not working in general, or just with your adjustments for the single channel?

**Comment by kevinhughes27** [comment]
So interesting but grayscale didn't work very well at all for me. After #32 all that is required is adding `im = rgb2gray(img)` to the start of `resize_image` and then the model changes from @millerda's fork.

Perhaps @bzier can doubly confirm but then we should probably just close this.

**Comment by millerda** [comment]
@bzier It wasn't working in general before I started trying to make adjustments for the single channel. Once I get home I can open a separate issue with the error output.
